src/dev/i18n/extract_code_messages.js

```diff
@@ -35,12 +35,20 @@ import { extractIntlMessages, extractFormattedMessages } from './extract_react_m
  *
  * Example: `intl.formatMessage({ id: 'message-id', defaultMessage: 'Message text' });`
  */
-function isIntlFormatMessageFunction(node) {
+export function isIntlFormatMessageFunction(node) {
```

Comment:
maybe let's keep this method private and test `extractCodeMessages`, what do you think?